SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -101,17 +97,7 @@ void USpatialWorkerConnection::ConnectToReceptionist(bool bConnectAsClient)
 	}
 
 #if WITH_EDITOR
-	const bool bSingleThreadedServer = !bConnectAsClient && (GPlayInEditorID > 0);
-	const int32 FirstServerEditorID = 1;
-	if (bSingleThreadedServer)
-	{
-		if (GPlayInEditorID == FirstServerEditorID)
-		{
-			WorkerController.InitWorkers(ReceptionistConfig.WorkerType);
-		}
-
-		ReceptionistConfig.WorkerId = WorkerController.WorkerIds[GPlayInEditorID - 1];
-	}
+	ReceptionistConfig.WorkerId = SpatialGDKServices::InitWorkers(ReceptionistConfig.WorkerType, bConnectAsClient);
```

Comment:
We only want to set this if it's single threaded.

Reply:
I guess it'll fall into the IsEmpty section regardless.

Reply:
Fixed.